server/odc-service/src/main/java/com/oceanbase/odc/service/collaboration/project/ProjectService.java

```diff
@@ -256,6 +260,9 @@ public Project setArchived(Long id, @NotNull SetArchivedReq req) throws Interrup
         if (!req.getArchived()) {
             throw new BadRequestException("currently not allowed to recover projects");
         }
+        if (scheduleService.hasEnabledScheduleInProject(id)) {
```

Comment:
may also audo disable for shedule while related project archived.

Reply:
check the project status during the next scheduling